README.rst

```diff
@@ -71,3 +71,45 @@ You can filter tests based on the attributes. There is a attribute named ``test_
 For running ``webidentity_test`` you'll need have Keycloak running.
 
 In order to run any STS test you'll need to add "iam" section to the config file. For further reference on how your config file should look check ``s3tests.conf.SAMPLE``.
+
+========================
+ IAM policy tests
+========================
+
+This is a set of IAM policy tests. Tests use the Boto2 and Boto3 libraries.
```

Comment:
I think these tests use only boto3, like you have mentioned at the end. Otherwise the README looks fine.

Reply:
Yes, I will correct that.
Thank you

Reply:
looks fine now.